cpp/src/barretenberg/stdlib/encryption/ecdsa/ecdsa_impl.hpp

```diff
@@ -13,6 +13,11 @@ bool_t<Composer> verify_signature(const stdlib::byte_array<Composer>& message,
 {
     Composer* ctx = message.get_context() ? message.get_context() : public_key.x.context;
 
+    // Check if revovery id v is either 27 ot 28.
+    // TODO(Suyash): check with Zac/Kesha/Cody.
+    field_t<Composer>(sig.v).assert_is_in_set({ field_t<Composer>(27), field_t<Composer>(28) },
```

Comment:
What is the meaning of this condition? Is it documented somewhere?

Reply:
I have added detailed comment explaining this (also added references). Resolving this.